style.css

```diff
@@ -0,0 +1,460 @@
+  transition: 0.3s;
+}
+
+.hover_red:hover {
+  background-color: var(--hover-red);
+  color: var(--hover-bright-red);
+  border-radius: 50px;
+  padding: 5px;
+  transition: 0.3s;
+}
+
+.hover_content:hover {
+    background-color: rgba(8,8,8,255);
+    transition: 0.3s;
+}
+
+.hover_underline:hover {
+  text-decoration: underline;
+}
+
+.hover_content * {
+    background-color: transparent;
+}
+/* SCROLL BEHAVIOUS and HOVER ends here */
+
+/* COLOURS and FONT SIZES starts here*/
+html {
+  --twitter-blue: #1d9bf0;
+  --dark-grey: #15181c;
+  --grey-: #6e767d;
+  --light-grey: #eff3f4;
+  --hover-square-grey: #1c1f23;
+  --hover-blue: rgba(10, 23, 31, 255);
+  --hover-bright-blue: rgba(26, 136, 210, 255);
+  --hover-green: rgba(7, 26, 20, 255);
+  --hover-bright-green: rgba(4, 174, 115);
+  --hover-red: rgba(32, 9, 20, 255);
+  --hover-bright-red: #d0156b;
+}
+
+h1 {
+  font-size: 20px;
+  font-weight: bold;
+  opacity: 100%;
+  margin: 0;
+}
+
+h2 {
+  font-size: 20px;
+  font-weight: 100;
+  margin-left: 15px;
+}
+
+h3 {
+  font-size: 20px;
+  font-weight: bold;
+}
+
+h4 {
+  font-size: 14px;
+}
+
+p {
+  font-size: 14px;
+}
+
+.show_more {
+  color: var(--twitter-blue);
+  font-size: 14px;
+  padding: 16px 16px 16px 16px;
+}
+
+.at_account {
+  font-size: 14px;
+  font-weight: lighter;
+  color: var(--grey-);
+}
+
+.sm_grey {
+  font-size: 13px;
+  color: var(--grey-);
+}
+/* PRIMARY COLOURS and FONT SIZES ends here*/
+
+/* MAIN GRID SETUP starts here */
+.container {
+  display: grid;
+  grid-template-columns: auto 250px 600px 370px auto;
+  gap: 5px;
+  margin: 0;
+  height: 100vh;
+}
+/* MAIN GRID SETUP starts here */
+
+/* LEFT NAV BAR starts here */
+.nav {
+  display: grid;
+  grid-template-rows: 1fr auto;
+  padding: 5px;
+  max-height: 100vh;
+}
+
+.nav-items {
+  height: auto;
+  margin: 10px 0px 5px 25px;
+  display: grid;
+  align-items: center;
+  justify-content: left;
+}
+
+.hover_round {
+    display: grid;
+    grid-template-columns: auto auto;
+    padding: 10px;
+    border-radius: 100px;
+  }
+
+.logo {
+  width: 30px;
+  height: 30px;
+  margin: 10px 0px 0px 40px;
+}
+
+.large_tweet {
+  background-color: var(--twitter-blue);
+  width: 200px;
+  height: 45px;
+  border-radius: 50px;
+  display: grid;
+  place-items: center;
+  font-size: 15px;
+  font-weight: bold;
+  margin-left: 20px;
+  margin-top: 15px;
+}
+
+.account {
+  display: grid;
+  grid-template-columns: auto 1fr auto;
+  align-items: center;
+  padding: 0px;
+}
+
+.account-details {
+  display: grid;
+  align-items: center;
+  row-gap: 3px;
+}
+
+.account-icon {
+  width: 45px;
+  height: 45px;
+  background-color: #c2185b;
+  border-radius: 50px;
+  margin: 10px;
+}
+
+.icon-right-padding {
+  margin-right: 10px;
+}
+/* LEFT NAV BAR ends here */
+
+/* MAIN - HEADER and STATUS bar - starts here */
+main {
+  display: grid;
+  grid-template-rows: 56px 1fr;
+  border-left: 2px solid var(--dark-grey);
+  border-right: 2px solid var(--dark-grey);
```

Comment:
could use - 
border: 0 2px solid var(--dark-grey);

Reply:
I always forget that. You are right, I try to use that when I remember.